datafusion/substrait/tests/cases/consumer_integration.rs

```diff
@@ -32,9 +32,51 @@ mod tests {
     use std::io::BufReader;
     use substrait::proto::Plan;
 
+    async fn register_csv(
+        ctx: &SessionContext,
+        table_name: &str,
+        file_path: &str,
+    ) -> Result<()> {
+        ctx.register_csv(table_name, file_path, CsvReadOptions::default())
+            .await
+    }
+
+    async fn create_context_tpch2() -> Result<SessionContext> {
+        let ctx = SessionContext::new();
+
+        let registrations = vec![
+            ("FILENAME_PLACEHOLDER_0", "tests/testdata/tpch/part.csv"),
```

Comment:
where do the `FILENAME_PLACEHOLDER_x` names come from? Might be nicer to have the proper names there instead (part, supplier, partsupp, ...), then there wouldn't be need to have separate contexts for each query?

Reply:
It comes from the query_2.json since we need those generated json files https://github.com/substrait-io/consumer-testing/tree/main/substrait_consumer/tests/integration/queries/tpch_substrait_plans

Reply:
Ah it's already there :/, then I guess better keep it like this!